### DISCUSSION TITLE
Some keyboard keys is not working when starting SEB browser

### DISCUSSION THREAD

**Discussion by jazzmango** [SafeExamBrowser/seb-win-refactoring#994] [General]
After launching SEB browser, some keyboard keys is not working, P, o, and space mostly. This seems to happen randomly. When we log out and close the SEB browser, the keyboard keys is working again. Anyone with similar issue?

**Comment by dbuechel** [comment]
SEB does not block individual alphanumerical keys nor the space bar. This may be a hardware or web application issue.

**Comment by jazzmango** [comment]
To reproduce this issue. Start SEB allow word as application. start word from SEB and press windows + L key, then the p l and some other keys is not responding. Close SEB and the keys working again

**Reply by dbuechel** [reply]
Thanks for the instructions, we'll have a look at at it.

**Reply by dbuechel** [reply]
This is actually a known issue, please refer to https://github.com/SafeExamBrowser/seb-win-refactoring/issues/305#issuecomment-1155196763.
